src/components/select-utils.js

```diff
@@ -35,7 +36,8 @@ export const joinPostData = state => postId => {
     const author = state.users[comment.createdBy]
     const isEditable = (user.id === comment.createdBy)
     const isDeletable = (user.id === post.createdBy)
-    return { ...comment, ...commentViewState, user: author, isEditable, isDeletable }
+    const compensateScroll = commentId === state.compensateScroll.commentId
```

Comment:
It's quite misleading, from its name I thought it's a function until I found this line. I'd suggest `needsScrollCompensation` or similar.